smt2parser/src/concrete.rs

```diff
@@ -33,78 +33,86 @@ pub struct Symbol(pub String);
 #[derive(Debug, PartialEq, Eq, Clone, Hash, Ord, PartialOrd, Serialize, Deserialize)]
 pub struct Keyword(pub String);
 
-/// Concrete identifier.
-pub type Identifier = crate::visitors::Identifier<Symbol>;
+pub use crate::visitors::Identifier;
```

Comment:
I'm now wondering if we should remove the `pub` at some point.

Reply:
> I'm now wondering if we should remove the `pub` at some point.


I made it `pub` to minimize breakage from dependencies referencing `smt2parser::concrete::Identifier` (and similar for the types in the `pub use` below). What do you think about moving the types defined in `visitors` (e.g. `Identifier`, `AttributeValue`, `DatatypeDec`, `FunctionDec`) to `concrete` now that there's now a single type for each?